docs/DeveloperGuide.md

```diff
@@ -262,30 +262,182 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
-    Use case ends.
-
-**Extensions**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* 3a. The given index is invalid.
-
-    * 3a1. AddressBook shows an error message.
-
-      Use case resumes at step 2.
+(For all use cases below, the **System** is `Serenity` and the **Actor** is the `User`, unless specified otherwise)
+
+**Setting Up**
+```
+UC01: Set up tutorial group
+
+System: Serenity
+Actor: User
+
+Guarantees:
+    - The tutorial group is added to the tutorial group list upon successful command.
+    - Students are added to the student list in the respective tutorial groups upon successful command.
+
+MSS:
+    1. User chooses a csv file to upload.
+    2. User adds the csv file in the same folder as the JAR file.
+    3. Serenity reads the csv file.
+    4. Serenity adds the tutorial groups and students to the respective lists.
+Use case ends.
+```
+
+**Attendance Taking**
+```
+UC02: For a tutorial group, mark all students present for a lesson
+
+System: Serenity
+Actor: User
+
+Preconditions: Tutorial groups and students have been set up
+Guarantees:
+    - Each student is marked present for a lesson upon successful command.
+
+MSS:
+    1. User requests to mark all students from a specific tutorial group present for a lesson.
+    2. User receives a confirmation message.
+    3. System shows the updated attendance list for the lesson.
+Use case ends.
+
+Extensions:
+    1a. Incomplete details are given.
+        1a1. System shows an error message.
+        Use case resumes at step 2.
```

Comment:
I interpreted step 2 as confirmation message (success message); is this what you intended? To redirect to 2 if there is incorrect input? 

(Because the google docs says resume at step 1)

Applies to all the other use cases too